src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag

```diff
@@ -25,7 +25,13 @@
         </div>
         <a id="frComment-visibility-options-trigger-${divId}"
            class="btn btn-sm btn-info pull-right"
-           onclick="toggleVisibilityEditForm(${divIdAsJsParams})">
+            <c:if test="${isEditForm}">
+                onclick="toggleVisibilityEditForm(${divIdAsJsParams})">
+            </c:if>
+            <c:if test="${isAddForm}">
+                onclick="toggleVisibilityAddForm(${divIdAsJsParams})">
+            </c:if>
+        >
```

Comment:
I mean the `>` on line 34, the `<a>` tag is closed on either line 29 or line 32 (`onclick="toggle...">`) already. The html changes on `instructorCommentsPageAddFrc.html` contains a new `&gt;`.